src/portal/setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
+universal=1
```

Comment:
These 2 lines can be removed as we only support python 3 now so the pakcage is no longer universal.